sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide.md

```diff
@@ -42,210 +38,208 @@ In the case of Service Bus, the modern client libraries have packages and namesp
 
 ### Client hierarchy
 
-In the interest of simplifying the API surface we've made a single top level client called `ServiceBusClient`, rather than one for each of queue, topic and subscription:
+In the interest of simplifying the API surface we've made a single top level client called `ServiceBusClient`, rather than one for each of queue, topic, subscription and session. This acts as the single entry point in contrast with six different entry points in the previous library. You can create senders and receivers off of this client to the queue/topic/subscription/session of your choice and start sending/receiving messages.
```

Comment:
nit:  `in the previous library` doesn't seem to add much and could be interpreted as pointing fingers at the old library.

Reply:
I do want to show the difference in the number of entry points... Updated in https://github.com/Azure/azure-sdk-for-net/pull/14166/commits/a02534f2b496a08d65a71effcf9e7d1de7136f1c